README.md

```diff
@@ -0,0 +1,330 @@
+```
+
+#### Resolution of artifacts as collection
+```java
+List<File> files = Maven.resolver().resolve("G:A:V")
+    .withTransitivity().asList(File.class);
+```
+
+#### Exclude specific dependencies while resolving an artifact
+```java
+File file = Maven.resolver().addDependencies(
+    MavenDependencies.createDependency("G:A:V", ScopeType.COMPILE, false,
+      MavenDependencies.createExclusion("G:B"))).resolve().withTransitivity().asFile();
+```
+
+#### Control Resolution Results with Strategy
+```java
+File file = Maven.resolver().resolve("G:A:V").using(new RejectDependenciesStrategy(false, "G:B")).asFile();
+```
+
+#### ShrinkWrap Resolver Strategies
+
+In ShrinkWrap Resolver, strategies allow you to modify the dependency graph and change the default behavior for resolving transitive dependencies.
+
+By default, Maven and ShrinkWrap Resolver do not resolve dependencies in provided and test scope, and skip optional dependencies.
+
+#### Available Strategies:
+
+- **AcceptAllStrategy (Equals TransitiveStrategy):**
+  Accepts all dependencies of artifacts.
+
+- **AcceptScopesStrategy:**
+  Accepts only dependencies with defined scope types.
+
+- **CombinedStrategy:**
+  Combines multiple strategies with a logical AND operation.
+
+- **NonTransitiveStrategy:**
+  Rejects all transitive dependencies; only directly specified dependencies are accepted.
+
+- **RejectDependenciesStrategy:**
+  Rejects dependencies defined by G:A. By default, it is transitive, meaning all dependencies originating at G:A are removed. To change this behavior to reject defined dependencies but keep their descendants, use `RejectDependenciesStrategy(false, "G:A")`.
+
+- **TransitiveStrategy (Equals AcceptAllStrategy):**
+  Accepts all dependencies of artifacts.
+
+
+#### Control Sources of Resolution
+
+ShrinkWrap Resolvers allows you to specify artifact resolution sources. By default, it uses the classpath (Maven Reactor) and Maven Central repository.
+
+```java
+Maven.configureResolver().withClassPathResolution(false).resolve("G:A:V").withTransitivity().asFile();
+Maven.configureResolver().withMavenCentralRepo(false).resolve("G:A:V").withTransitivity().asFile();
+Maven.configureResolver().workOffline().resolve("G:A:V").withTransitivity().asFile();
+Maven.configureResolver().useLegacyLocalRepo(true).resolve("G:A:V").withTransitivity().asFile();
+```
+
+#### Specify settings.xml
+```java
+Maven.configureResolver().fromFile("/path/to/settings.xml")
+  .resolve("G:A:V").withTransitivity().asFile();
+
+Maven.configureResolver().fromClassloaderResource("path/to/settings.xml")
+  .resolve("G:A:V").withTransitivity().asFile();
+```
+
+<sub>**Warning:** ShrinkWrap Resolvers doesn't consume settings.xml specified on the command line or in the IDE. It reads settings.xml files at standard locations: `~/.m2/settings.xml` and `$M2_HOME/conf/settings.xml`, unless overridden in the API or via System property.</sub>
+
+#### Define Maven repositories manually
```

Comment:
The bit about overriding a repository with the same id should probably stay. Perhaps as a note as above?